x/emissions/module/rewards/scores.go

```diff
@@ -483,3 +522,45 @@ func EnsureAllWorkersPresentWithheld(
 
 	return values
 }
+
+// CalculateTopicInitialEmaScore calculates the initial EMA score for new participants in the next epoch
+// using the current active set statistics. The initial score is set below the active set threshold
+// by using the formula: lowestEmaScore - lambda * standardDeviationOfEmaScores
+func CalculateTopicInitialEmaScore(
+	ctx context.Context,
+	keeper keeper.Keeper,
+	activeScores []types.Score,
+) (alloraMath.Dec, error) {
+	// Get lambda parameter from module params
+	params, err := keeper.GetParams(ctx)
+	if err != nil {
+		return alloraMath.Dec{}, errors.Wrapf(err, "error getting module params")
+	}
+	lambda := params.LambdaInitialScore
+
+	// Calculate standard deviation of EMA scores in active set
+	var emaScores []alloraMath.Dec
+	lowestScore := activeScores[0]
```

Comment:
is there a chance that activeScores contains no elements? Should we check against this?

Reply:
That is checked on L544